opencti-platform/opencti-front/src/private/components/Dashboard.jsx

```diff
@@ -908,25 +905,47 @@ const CustomDashboard = ({ dashboard, timeField }) => {
     </Security>
   );
 };
+
+const changeDashboardPref = graphql`
+  mutation DashboardPrefMutation($input: [EditInput]!) {
+    meEdit(input: $input) {
+      dashboard_id
+    }
+  }
+`;
+
 const Dashboard = () => {
   const classes = useStyles();
-  const [view, saveView] = useViewStorage('view-dashboard');
-  const { dashboard = 'default', timeField = 'technical' } = view;
-  const handleChangeTimeField = (event) => saveView({ dashboard, timeField: event.target.value });
-  const handleChangeDashboard = (event) => saveView({ dashboard: event.target.value, timeField });
+  const { me: { dashboard_id: dashboardId } } = useContext(UserContext);
+  const [changeDashboard] = useMutation(changeDashboardPref);
+  const [localDashboardPreferences, saveLocalDashboardPreferences] = useLocalStorage('view-dashboard', {});
+  const { timeField = 'technical' } = localDashboardPreferences;
+  const handleChangeTimeField = (event) => saveLocalDashboardPreferences({ dashboard: dashboardId, timeField: event.target.value });
+  const handleChangeDashboard = (event) => {
+    changeDashboard({
+      variables: {
+        input: [{
+          key: 'dashboard_id',
+          value: (event.target.value === 'default' ? [null] : event.target.value),
+        }],
+      },
+    });
+
+    saveLocalDashboardPreferences({ dashboard: event.target.value, timeField });
```

Comment:
My intention is to explicit that the *save* operation is not an update to the server, which can be confusing in this context because we are doing both (the dashboard is updated but the timefield is saved locally). I used this verbosity as a way to have this distinction.

That being said, I may have missed better way to handle this problem, in which case, I welcome suggestion.